### PR TITLE
Add managed accounts API

### PR DESCRIPTION
**Pull request by plfx**
- createManagedAccount
- getManagedAccounts

The singular `getManagedAccount` function is not necessary at this time. @medikoo In order to check for managed accounts, you can fetch the list of managed accounts. This list will contain at most one account in `SUCCEEDED` state.

### CODE REVIEW

File: src/managedAccounts/getManagedAccounts.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import fetch from '../fetch'
+import platformConfig from '../config'
+
+export default async ({ accessKey, tenant }) => {
+  const response = await fetch(`${platformConfig.backendUrl}tenants/${tenant}/managedAccounts`, {
+    method: 'GET',
```
Comment (medikoo) [inline src/managedAccounts/getManagedAccounts.js]: API Documentation states it's a `POST`, but I take it's an error on documentation side

File: src/managedAccounts/createManagedAccount.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import fetch from '../fetch'
```
Comment (medikoo) [inline src/managedAccounts/createManagedAccount.js]: If this is intended to purely support creation of AWS free preview managed account, I think it should be named `createAwsPreviewManagedAccount`, if not I think there should be an option to indicate that we're after this specific account.

At least leaving it that way, may introduce issues in future (when e.g. we decide to support creation of other managed accounts that way)

File: src/managedAccounts/createManagedAccount.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+import platformConfig from '../config'
+
+export default async ({ accessKey, captchaToken, tenant }) => {
+  const response = await fetch(`${platformConfig.backendUrl}tenants/${tenant}/managedAccounts`, {
+    method: 'POST',
+    headers: { Authorization: `bearer ${accessKey}`, 'X-Captcha-Token': captchaToken },
```
[the remark this answers is not in the document]
Comment (dschep) [reply src/managedAccounts/createManagedAccount.js]: website->CLI comms are already done, see login. It's done by virtue of the CLI starting a local server and the website redirecting to it with query params and then it redirects back to the website

[the remark this answers is not in the document]
Comment (medikoo) [reply src/managedAccounts/createManagedAccount.js]: > website->CLI comms are already done, see login

I know, but I know it also doesn't work well :) I heard it doesn't work with Safari, and we have problems with ports (altough we can be smart and shuffle the port until we find the free one).

I think it demands further investigation whether we can real have such setup really 100% bulletproof. Problem is that what we're practicing currently, is not, and if we cannot work out setup that will work for 100%, we should seek alternatives.

Also I believe `login` should work same as registration, without involving a browser, it'll give nicer user experience. It's already weird that _registration_ (which also logs user in) doesn't need browser, while login does.

**Comment by plfx** [comment]
Closing this PR since the feature was cancelled

**Pull request closed without merging**